### PR TITLE
Github action changes for deploying nestjs backend

### PR DESCRIPTION
**Pull request by pranavxc**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [x] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: scripts/upgradeNcGui.js [hunk 1]
```diff
@@ -35,11 +35,11 @@ const bumbVersionAndSave = () => {
 if (process.env.targetEnv === 'DEV') {
     // replace nc-lib-gui by nc-lib-gui-daily if it is nightly build / pr release
     const filePaths = [
-        path.join(__dirname, '..', 'packages', 'nocodb', 'Dockerfile'),
-        path.join(__dirname, '..', 'packages', 'nocodb', 'litestream', 'Dockerfile'),
-        path.join(__dirname, '..', 'packages', 'nocodb', 'package.json'),
-        path.join(__dirname, '..', 'packages', 'nocodb', 'README.md'),
-        path.join(__dirname, '..', 'packages', 'nocodb', 'src', 'lib', 'Noco.ts'),
+        path.join(__dirname, '..', 'packages', 'nocodb-nest', 'Dockerfile'),
+        path.join(__dirname, '..', 'packages', 'nocodb-nest', 'litestream', 'Dockerfile'),
```
Comment (wingkwong) [inline scripts/upgradeNcGui.js:39]: this file doesn't exist

Comment (pranavxc) [reply scripts/upgradeNcGui.js:39]: I will add that one as well :+1:

**Comment by wingkwong** [comment]
@pranavxc all green now. please take a final look.

**Comment by pranavxc** [comment]
> @pranavxc all green now. please take a final look.

All looks good now 👍🏽

**Pull request merged**